.github/renovate/eslint-base.json5

```
@@ -0,0 +1,19 @@
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  extends: ["github>eslint/workflows//.github/renovate/base.json5"],
```

Can we just use a local file path instead?
```suggestion
  extends: ["base.json5"],
```

---

not sure if it can work as expected - it may be seen as built-in preset. I will change it to `./base.json5` to see if it's working.

---

FYI, I've also looked at the official documentation, but it seems there's no way to use a local path directly. 🤔

https://docs.renovatebot.com/config-presets/#github